app/views/trainees/new.html.erb

```diff
@@ -11,9 +11,8 @@
 
       <%= f.govuk_error_summary %>
 
-      <h1 class="govuk-heading-l">
-        Add a trainee
-      </h1>
+      <%= render_component PageHeading::View.new(title: "trainees.new", 
+                                                 errors: @trainee.errors) %>
```

Comment:
I'm not sure why this would need to know about errors - errors have no bearing on the page headings.